README.md

```diff
@@ -1534,14 +1541,21 @@ Language: English
 
 [__IntellectualSites__](https://discord.com/invite/KxkjDVg) [<img height="16px" width="16px" alt="Official Badge" src="images/badges/official.webp">](badges.md#official-identification-badge) [<img height="16px" width="16px" alt="Homepage URL" src="images/badges/homepage.webp">](https://intellectualsites.com/) [<img height="16px" width="16px" alt="Git Repository" src="images/badges/git.webp">](https://github.com/IntellectualSites) \
 Notable Channels: `#fawe-support`, `#fawe-developer-chat`, `#ps-developer-chat`, `#hv-general-chat`, `#is-web`, `#programming` \
-Language: English
+Language: English <br/><br/>
 
 <img align="left" height="94px" width="94px" alt="Server Icon" src="images/server_icons/linuxgsm_.webp" />
 
 [__LinuxGSM__](https://discord.com/invite/UM4VTpK) [<img height="16px" width="16px" alt="Official Badge" src="images/badges/official.webp">](badges.md#official-identification-badge) [<img height="16px" width="16px" alt="Homepage URL" src="images/badges/homepage.webp">](https://linuxgsm.com/) [<img height="16px" width="16px" alt="Git Repository" src="images/badges/git.webp">](https://github.com/GameServerManagers      ) \
 Notable Channels: `#linuxgsm`, `#linux`, `#game-server`, `#general`, `#battlestations` \
+Language: English <br/> <br/>
+
+
+<img align="left" height="94px" width="94px" alt="Server Icon" src="https://pluckyporcupinegame.boutique/wp-content/uploads/2020/03/raylib_256x256.png" />
+
+[__Raylib__](https://discord.com/invite/VkzNHUE) [<img height="16px" width="16px" alt="Reddit Badge" src="images/badges/reddit.webp">](https://www.reddit.com/r/raylib/) [<img height="16px" width="16px" alt="Homepage URL" src="images/badges/homepage.webp">](https://www.raylib.com/) [<img height="16px" width="16px" alt="Git Repository" src="images/badges/git.webp">](https://github.com/raysan5/raylib) \
```

Comment:
Probably raylib should be moved to a separate PR...

Reply:
Yeah @mhxion should accept the PL